recipes/loguru/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+
+required_conan_version = ">=1.54.0"
+
+
+def _int(flag):
+    if flag is not None and flag:
+        return "1"
+    else:
+        return "0"
+
+
+class LoguruConan(ConanFile):
+    name = "loguru"
+    version = "2.1.0"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = 'CMakeLists.txt'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/emilk/loguru"
+    license = "Unlicense"  # Public domain
+    description = "Loguru is a C++11 logging library."
+
+    options = {
+        "use_fmtlib": [True, False],
+        "shared": [True, False],
+        "catch_sigabrt": [True, False],
+        "verbose_scope_endings": [True, False],
+        "redefine_assert": [True, False],
+        "with_streams": [True, False],
+        "with_fileabs": [True, False],
+        "with_rtti": [True, False],
+        "replace_glog": [True, False],
+
+        "scope_text_size": "ANY",       # int
+        "filename_width": "ANY",        # int
+        "threadname_width": "ANY",      # int
+        "scope_time_precision": "ANY",  # int
```

Comment:
```suggestion
        "scope_text_size": [None, "ANY"],
        "filename_width": [None, "ANY"],
        "threadname_width": [None, "ANY"],
        "scope_time_precision": [None, "ANY"],
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#any-special-value